ARM/STM32/svd/stm32f40x/stm32_svd-adc.ads

```diff
@@ -718,45 +719,45 @@ package STM32_SVD.ADC is
    --  Analog-to-digital converter
    type ADC1_Peripheral is record
       --  status register
-      SR    : SR_Register;
+      SR    : aliased SR_Register;
```

Comment:
What is the rational for this?

Also, the current version of SVD2Ada doesn't generate the output file. You must have use a custom SVD2Ada.